beacon-chain/blockchain/service_test.go

```diff
@@ -183,21 +183,84 @@ func TestFaultyStop(t *testing.T) {
 
 	chainService.Start()
 
-	chainService.chain.SetActiveState(types.NewActiveState(nil))
+	chainService.chain.SetActiveState(types.NewActiveState(nil, make(map[*common.Hash]*types.VoteCache)))
 	err = chainService.Stop()
 	if err == nil {
 		t.Errorf("chain stop should have failed with persist active state")
 	}
 
-	chainService.chain.SetActiveState(types.NewActiveState(&pb.ActiveState{}))
+	chainService.chain.SetActiveState(types.NewActiveState(&pb.ActiveState{}, make(map[*common.Hash]*types.VoteCache)))
 	chainService.chain.SetCrystallizedState(types.NewCrystallizedState(nil))
 	err = chainService.Stop()
 	if err == nil {
 		t.Errorf("chain stop should have failed with persist crystallized state")
 	}
 }
 
-func TestProcessingBadBlock(t *testing.T) {
+func TestProcessingBlock(t *testing.T) {
```

Comment:
This isn't testing anything, just running through code. We should just get rid of this.

Reply:
Ya you are right. Deleted